### ISSUE TITLE
[simprod-scripts] UseGPU vs UseGPUs (Trac #1815)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1807]
Clean up the parameter names of the modules/segments by settling on singular vs plural for UseGPU(s).  It looks like most of them use plural, while the clsim and ppc modules use singular.

We might want to check for other problems like this, where parameter names are almost the same between modules.

Migrated from https://code.icecube.wisc.edu/ticket/1815
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:35", 
    "description": "Clean up the parameter names of the modules/segments by settling on singular vs plural for UseGPU(s).  It looks like most of them use plural, while the clsim and ppc modules use singular.\n\nWe might want to check for other problems like this, where parameter names are almost the same between modules.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067215093672", 
    "component": "combo simulation", 
    "summary": "[simprod-scripts] UseGPU vs UseGPUs", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-08-11T15:01:44", 
    "milestone": "", 
    "owner": "juancarlos", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160812T16:48:05: 
* juancarlos changed resolution from "" to "fixed"
* juancarlos changed status from "new" to "closed"
* juancarlos commented:

> In changeset 149217:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="149217"
> Closes #1807
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:35: 
* olivas commented:

> Milestone IceSim 5.1 deleted


* olivas changed milestone from "IceSim 5.1" to ""
